### ISSUE TITLE
Release to support MacOS Catalina?

### ISSUE THREAD

**Issue by swiftcoder** [gfx-rs/wgpu-rs#110]
The newest version of wgpu on crates.io is 0.3.0, which only seems to build against winit version 0.20.0-alpha3. Unfortunately, Mac OS Catalina breaks compilation of that version of winit, so one needs winit 0.20.0-alpha4.

Everything works fine if I point my local build at the wgpu master branch. However I can't really release anything on Mac until a public wgpu release supports the newer version of winit.

**Comment by kvark** [comment]
wgpu-rs isn't written against `winit`, it's just the examples. wgpu-rs only cares about raw-window-handle, which is perfectly supported by winit-0.20.0-alpha4. Moreover, the examples today already use winit-0.20.0-alpha4, so I don't think there is anything to do here, is there?

**Comment by rukai** [comment]
What swiftcode is saying is:
The current release of wgpu (0.3.0) uses raw_window_handle 0.1 which can only use winit 0.20.0-alpha3.
They are requesting a new release of wgpu (0.4.0) so that they can use later winit versions via crates.io

**Comment by swiftcoder** [comment]
Yep. The problem is that you always end up with both `raw-window-handle v0.1.2` and `raw-window-handle v0.3.1` in your dependency tree, leading to the dreaded mismatched types error:

```
error[E0308]: mismatched types
  --> examples/triangle.rs:17:47
   |
17 |         let surface = instance.create_surface(window.raw_window_handle());
   |                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `raw_window_handle::RawWindowHandle`, found a different enum `raw_window_handle::RawWindowHandle`
   |
   = note: expected type `raw_window_handle::RawWindowHandle` (enum `raw_window_handle::RawWindowHandle`)
              found type `raw_window_handle::RawWindowHandle` (enum `raw_window_handle::RawWindowHandle`)
note: Perhaps two different versions of crate `raw_window_handle` are being used?
```

**Comment by kvark** [comment]
The ETA for wgpu-0.4 release is this weekend or the start of the week. Hold tight!

**Comment by kvark** [comment]
Technically you can point a cargo patch on the git master today and get going, until the release is made.

**Comment by rukai** [comment]
This can be closed now, 0.4 is released
